helpers/search_processing_kendra_quickstart.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+set -o errexit
+set -o errtrace
+set -o pipefail
+set -o nounset
+
+# Some useful constants
+readonly DOCKER_IMAGE_TAG="opensearch-with-ranking-plugin"
+readonly OPENSEARCH_VERSION="2.4.0"
+
+function print_help() {
+  echo "Usage: $0 -p <execution_plan_id> [-r <region>] [-e <kendra_ranking_endpoint>]"
+  echo "  -p | --execution-plan-id          The ID returned from Kendra Intelligent Ranking service"
```

Comment:
That's a good idea!  I'll add that in my next revision.

Reply:
Another thought.. Since creating the execution plan will require the very latest version of the CLI (with support for kendra-ranking) can the script defend against an old version too? ie check the version, and if kendra-ranking isn't supported, then output link to cli update instructions?  That'll cover one more thing that could go wrong :)

Reply:
I added support for creating the execution plan.

It checks for kendra-ranking support based on the exit code for `aws kendra-ranking help` (which works -- I tried with and without the kendra-ranking service model added to my CLI).

To avoid bringing up the OpenSearch containers before the execution plan is ready, it sits in a loop describing the execution plan every 30 seconds until the status becomes "ACTIVE" before proceeding.